src/test/java/org/takes/facets/auth/codecs/CcStrictTest.java

```diff
@@ -39,24 +40,33 @@
     /**
      * CcStrict can block empty urn.
      * @throws IOException If some problem inside
-     * @todo #877:30min replace all org.junit.Test occurrences in the project
-     *  by JUnit 5 equivalent org.junit.jupiter.api.Test and replaces
-     *  all expected Exceptions by Assertions.assertThrows. Check
-     *  org.takes.facets.auth.codecs.CcAesTest or
-     *  org.takes.facets.auth.codecs.CcSaltedTest as examples.
      */
-    @Test(expected = DecodingException.class)
+    @Test
     public void blocksEmptyUrn() throws IOException {
-        new CcStrict(new CcPlain()).encode(new Identity.Simple(""));
+        Assertions.assertThrows(
+            DecodingException.class,
```

Comment:
@marceloamadeu Please add a message informing the reason of the assertion failure